src/MSA.BuildingBlocks.CosmosDbMigration/Abstractions/BaseContainerMigration.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.Azure.Cosmos;
+using System;
+using System.Collections.Generic;
+using System.Dynamic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MSA.BuildingBlocks.CosmosDbMigration.Abstractions;
+
+public abstract class BaseContainerMigration
+{
+
```

Comment:
empty line and other code style issues

Reply:
In order to ensure code style consistency across the project, I highly recommend integrating an .editorconfig file into our solution.

Reply:
In order to ensure code style consistency across the project, I highly recommend integrating an .editorconfig file into our solution.

Reply:
good point. I'll create an issue